Libs/DICOM/Widgets/ctkDICOMAppWidget.cpp

```diff
@@ -527,8 +539,12 @@ void ctkDICOMAppWidget::openQueryDialog()
 {
   Q_D(ctkDICOMAppWidget);
 
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.